client/watch.html

```diff
@@ -731,7 +735,7 @@
 </template>
 
 <template name="list_item_context_section">
-  <div class="list-item-context-plus-annotation" data-context-id="{{_id}}">
+  <div class="list-item-context-plus-annotation" data-context-id="{{_id}}" style="{{#unless showContext}} display:none;{{/unless}}">
```

Comment:
instead of `display: none`, you're probably better off not having it in the DOM at all (i.e. putting the `if` outside the whole div). maps, for example, render poorly forever when they come in as `display: none` I believe.